src/theme/components/customTable.ts

```diff
@@ -1,8 +1,28 @@
+// What we might expect users to pass for the color "white". This is
+// temporary until we have a better color contrast checker.
+const whiteColors = [
+  "#fff",
+  "#ffffff",
+  "#FFF",
+  "#FFFFFF",
+  "white",
+  "var(--nypl-colors-ui-white)",
+];
 const baseStyle = ({
   columnHeadersTextColor,
   showRowDividers,
   useRowHeaders,
 }) => ({
+  // Just so we don't let row headers to display as white. Only the
+  // thead should have white text color with any color background.
+  tbody: {
+    th: {
+      color:
```

Comment:
I misunderstood how this was going to be addressed.  The row headers should simply stay black all the time and we should only allow the color of the column headers to be changed.